src/app/visual/buttons/buttons-demo.component.html

```diff
@@ -78,6 +78,45 @@ <h2>
       with some other text.
     </div>
 
+    <div
+      class="sky-buttons-demo-screenshot-inline"
+    >
+      <div
+        class="sky-input-group"
+        style="width:400px;"
+      >
+        <input
+          class="sky-form-control"
+          type="text"
+        >
+        <div
+          class="sky-input-group-btn"
+        >
+          <button
+            aria-haspopup="dialog"
+            class="sky-btn sky-btn-default"
+            type="button"
+          >
+            Input
+          </button>
+          <button
+            aria-haspopup="dialog"
+            class="sky-btn sky-btn-default"
+            type="button"
+          >
+            group
+          </button>
+          <button
+            aria-haspopup="dialog"
+            class="sky-btn sky-btn-default"
+            type="button"
+          >
+            buttons
```

Comment:
I would use a Font Awesome icon for the content of each input group button as that will be the majority use case.